utils/strings/formatters.ts

```diff
@@ -0,0 +1,29 @@
+export function removeProtocolAndWwwIfPresent(url: string) {
+  const httpProtocolPrefix = 'http://';
+  const httpsProtocolPrefix = 'https://';
+  const wwwPrefix = 'www.';
+
+  let trimmedUrl = url;
+
+  if (trimmedUrl.startsWith(httpProtocolPrefix)) {
+    trimmedUrl = trimmedUrl.slice(httpProtocolPrefix.length);
+  }
+
+  if (trimmedUrl.startsWith(httpsProtocolPrefix)) {
+    trimmedUrl = trimmedUrl.slice(httpsProtocolPrefix.length);
+  }
+
+  if (trimmedUrl.startsWith(wwwPrefix)) {
+    trimmedUrl = trimmedUrl.slice(wwwPrefix.length);
+  }
+
+  return trimmedUrl;
+}
+
+export function capitalizeEveryWord(phrase: string) {
+  return phrase
+    .toLowerCase()
+    .split(' ')
+    .map((word) => word.charAt(0).toUpperCase() + word.slice(1))
+    .join(' ');
```

Comment:
Woah I wasn't aware that .charAt(1) returns "" for single letter or zero letter strings. Could we add a comment to document it. (Or ignore it, this might just be me)
```suggestion
  return phrase
    .toLowerCase()
    .split(' ')
    // Safely handles word with length 0 or 1 as well.
    .map((word) => word.charAt(0).toUpperCase() + word.slice(1))
    .join(' ');
```